Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -220,63 +190,16 @@ public class Klaviyo: NSObject  {
      */
     public func trackEvent(event: String?, customerProperties: NSDictionary?, propertiesDict: NSDictionary?, eventDate: NSDate?) {
         
-        var eventName = event
-        // Set default track name if none provided
-        if (eventName == nil || eventName!.isEmpty) { eventName = "KL_Event" }
-
+        guard let eventName = event, !eventName.isEmpty else {
```

Comment:
why is event an optional in the first place? Is that just backward compatibility?